mw-webapp/src/logic/wayPage/reportsTable/DayReportsTable.tsx

```diff
@@ -15,6 +18,11 @@ interface DayReportsTableProps {
    * Way's uuid
    */
   wayUuid: string;
+
+  /**
+   * Way
+   */
+  way: WayPreview;
```

Comment:
git rid of this prop

Reply:
The same. Discussed and allowed to use this prop